sites/packworld/redirect-handler.js

```diff
@@ -0,0 +1,30 @@
+const redirect = (from, pattern) => pattern.exec(from);
+
+module.exports = ({ from }) => {
+  const digital = redirect(from, /\/sites\/default\/files\/digital_edition\/(.*)/);
+  if (digital && digital[1]) {
+    return { to: `https://digitaleditions.packworld.com/${digital[1]}` };
+  }
+
+  const s3 = redirect(from, /\/sites\/default\/files\/(.*)/);
+  if (s3 && s3[1]) {
+    return { to: `https://s3.us-east-2.amazonaws.com/pmg-production/Migrated+-+DO+NOT+USE/AW/${s3[1]}` };
+  }
+
+  const showcase = redirect(from, /^\/showcasesubmit/);
+  if (showcase && showcase[1]) {
+    return { to: 'https://showcasesubmit.packworld.com' };
+  }
+
+  const mundoHome = redirect(from, /^\/mundopmmi$/);
+  if (mundoHome && mundoHome[1]) {
+    return { to: 'https://www.mundopmmi.com' };
+  }
+
+  const mundo = redirect(from, /^\/mundopmmi\/.*/);
+  if (mundo && mundo[1]) {
+    return { to: `https://www.mundopmmi.com${from}` };
```

Comment:
This is going to redirect to `https://www.mundopmmi.com/mundopmmi/some/path/here` which is in correct

Reply:
Yes because Mundopmmi used to be part of Packworld. BK told them to redirect all traffic from that section to Mundo. Mondo has the redirects in their collection.  We might want to consider moving these to packworld so it's a single redirect but currently, they are broken so this is a stop gap.